README.md

```diff
@@ -141,6 +141,88 @@ const searchResult = await shopperSearch.productSearch({
 
 Invalid query parameters that are not a part of the API and do not follow the `c_` custom query parameter convention will be filtered from the request and a warning will be displayed.
 
+### Custom APIs
+
+The SDK supports calling [custom APIs](https://developer.salesforce.com/docs/commerce/commerce-api/guide/custom-apis.html) with a helper function, `callCustomEndpoint`.
+
+Example usage:
+
+```javascript
+import pkg from 'commerce-sdk-isomorphic';
+const { helpers } = pkg;
+
+const CLIENT_ID = "<your-client-id>";
+const ORG_ID = "<your-org-id>";
+const SHORT_CODE = "<your-short-code>";
+const SITE_ID = "<your-site-id>";
+
+// client configuration parameters
+const clientConfigExample = {
+  parameters: {
+    clientId: CLIENT_ID,
+    organizationId: ORG_ID,
+    shortCode: SHORT_CODE,
+    siteId: SITE_ID,
+  }
+};
```

Comment:
I don't feel strongly, but you're repeating yourself here:

```suggestion
const clientConfigExample = {
  parameters: {
    clientId: <your-client-id>,
    organizationId: "<your-org-id>",
    shortCode: "<your-short-code>",
    siteId: "<your-site-id>",
  }
};
```

Also, because you've chosen meaningful variable names, you don't really need the code comment.

Why do you pass `siteId` as a param to the client Config and then pass it again as a `parameter` later?

Reply:
That was a miss where I didn't use the siteId from the clientConfig, code has been updated to use siteId as query parameter if available